vslib/src/sai_vs_switch_BCM56850.cpp

```diff
@@ -185,6 +199,11 @@ static sai_status_t create_ports()
 
         CHECK_STATUS(vs_generic_set(SAI_OBJECT_TYPE_PORT, port_id, &attr));
 
+        attr.id = SAI_PORT_ATTR_OPER_STATUS;
+        attr.value.s32 = SAI_PORT_OPER_STATUS_UP;
```

Comment:
>SAI_PORT_OPER_STATUS_UP [](start = 25, length = 23)

isn't the default value of port oper status to be down, and when you see netlink messages you will set it to be the correct value.